### DISCUSSION TITLE
Aggregators

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#160] [Ideas]
Examples:
* DistroKids 
* CDBaby 
* [Horus Music](https://www.horusmusic.global/) 
* [DashGo](https://dashgo.com/login)
* [TuneCore](https://www.tunecore.com/)

**Comment by tshiunghan** [comment]
Like exploring how to get Mirlo to interface with aggregators?

**Comment by tshiunghan** [comment]
I looked into Distrokid and it looks like they don't have an API (i.e., there's no info about one in their support docs nor their distribution agreement) and they expect users to use their GUI to input metadata.

**Reply by simonv3** [reply]
Yeah, I think the idea is more to replicate what they're doing rather than integrating with them (for exactly that reason you mention).

**Comment by tshiunghan** [comment]
I found [this 2018 Medium post where the Distrokid founder talks about introducing more metadata types for uploads](https://news.distrokid.com/credits-34db783febf6). 

In passing, he mentions that the music industry standards body DDEX introduced [Recording Information Notification (RIN)](https://kb.ddex.net/implementing-each-standard/recording-information-notification-(rin)/). 

Have you already come across this? Do you think this could be helpful in defining metadata types for Mirlo?

**Reply by simonv3** [reply]
Nope, haven't looked at it at all. I would appreciate it if someone else could take on the initial research task of that. But also, I noted this as a discussion topic mainly because I didn't want it to get lost. I don't think this is MVP material.

**Comment by Cwpute** [comment]
I don't wee why you'd want Mirlo to interface with aggregators. Can you explain ?

**Reply by simonv3** [reply]
The idea is more to replicate aggregator behavior so that Mirlo could push music to Spotify etc like the aggregators do. It's a Can of Worms Though. That should probably be elaborated on in the original issue

**Reply by Cwpute** [reply]
Yeah, incredibly big can of worms 😬 one could say it's a bucket of snakes ! You'd be dealing with lots of new third-parties and legal stuff, and on top of that, be dealing with publishers people are probably trying to avoid in the first place coming to Mirlo !

**Comment by tshiunghan** [comment]
Spotify music distributors: https://artists.spotify.com/providers

Apple Music distributors: https://artists.apple.com/partners

Couldn't find a list for Google Play.

**Comment by nemobis** [comment]
DistroKid has an interesting approach. https://distrokid.com/

> We'll get your music into Spotify, Apple Music, TikTok, Pandora, Amazon, Instagram, YouTube, Tidal, iHeartRadio, Deezer, plus a ton of other stores & streaming services.
> Only €23.99 to upload unlimited songs & albums for a year.

Even for people interested in Mirlo's cooperative approach, breaking free from all the streaming services at once would probably be too much of a break. Allowing to optionally keep posting into some of these would make the transition less painful. The yearly price resembles a membership fee.

A feature I see used in the wild is what they call "hyperfollow", a linkstack of sorts which allows to link one canonical URL for a song or album and to let the visitor find a preview and links to streaming services or purchase options ([example](https://distrokid.com/hyperfollow/kuopus/suku-suru-jne)). Even if Mirlo doesn't do the distribution, it would be nice to become the canonical URL where you can find links to every other location, so that musicians can start linking a less exploitative landing page from their social media etc.

![Screenshot_20250420_111908](https://github.com/user-attachments/assets/d9041f10-fbff-4a28-8e02-048e2f85ffae)

**Comment by nemobis** [comment]
There's also https://orcd.co/ (example page: https://orcd.co/lempieloaavat ).
